rsa.go

```diff
@@ -85,7 +85,7 @@ func VerifyRecoverRSASignature(publicKey, signature []byte) ([]byte, error) {
 // - Parameter pkeyopt: A map of any algorithm specific control operations in string form
```

Comment:
This says `any` currently, but now it only supports `rsa_padding_mode` and `rsa_pss_saltlen`.  We should update this comment or maybe add support for the other possible operations; unfortunately it's not clear in the documentation what all the supported options are exactly.

Reply:
Yeah agreed, I was thinking in terms of this function as a generic solution for other use cases.  Main thing is making sure the documentation here is clear.

Reply:
I left the documentation unchanged because I added fallback logic to set any remaining pkeyopt that aren't specifically called out. We can add further exceptions as we onboard different documents and see different behavior.